[Term Entry] Data Science Data Distributions: Normal Distribution (#6264)
Codecademy/docs: content/data-science/concepts/data-distributions/terms/normal-distribution/normal-distribution.md

---
Title: 'Normal Distribution'
Description: 'A continuous probability distribution characterized by a bell-shaped curve that is symmetric around its mean.'
Subjects:
  - 'Data Science'
  - 'Data Visualization'
  - 'Machine Learning'
Tags:
  - 'Data Science'
  - 'Deep Learning'
CatalogContent:
  - 'learn-python-3'
  - 'paths/data-science'
---

> The **Normal Distribution**, also known as the Gaussian Distribution, is one of the most important probability distributions in statistics and data science. It is defined by two parameters:

Reviewer: ```suggestion
The **Normal Distribution**, also known as the Gaussian Distribution, is one of the most important probability distributions used for continuous data. It is defined by two parameters:
```
